temporal-sdk/src/test/java/io/temporal/workflow/StartTest.java

```diff
@@ -53,18 +53,26 @@ public void testStart() {
     assertResult("func", WorkflowClient.start(stubF::func));
     Assert.assertEquals(
         "func", stubF.func()); // Check that duplicated start just returns the result.
-    WorkflowOptions options =
-        WorkflowOptions.newBuilder().setTaskQueue(testWorkflowRule.getTaskQueue()).build();
-    Test1ArgWorkflowFunc stubF1 =
-        testWorkflowRule.getWorkflowClient().newWorkflowStub(Test1ArgWorkflowFunc.class, options);
+  }
 
+  @Test
+  public void startOneArgsFuncWithDefault() {
+    // TODO why it doesn't work with external service?
```

Comment:
Does this need to be addressed in this PR?

Reply:
No, only this note is new, the code and exclusion of the external service were always there. Completely irrelevant to the nature of this PR. Left it for the future me mostly.